### PR TITLE
remove lodash dependency from rtp and common packages

### PR DESCRIPTION
**Pull request by koush**
I'm using the rtp sub package within another project and was looking to reduce dependencies for deployed size and supply chain attack surface area. I found that lodash would be trivial to remove (64kb in bundle).

### CODE REVIEW

**Comment by koush** [comment]
I believe this hang is an error in the tests per our previous conversations?

File: packages/rtp/src/rtcp/rtpfb/twcc.ts [hunk 1]
```diff
@@ -313,14 +312,20 @@ export class StatusVectorChunk {
...
+    }
+
     switch (symbolSize) {
       case 0:
-        range(6).forEach((_, i) => symbolList.push(getBit(data[0], 2 + i, 1)));
-        range(8).forEach((_, i) => symbolList.push(getBit(data[1], i, 1)));
+        range(6, (i) => symbolList.push(getBit(data[0], 2 + i, 1)));
+        range(8, (i) => symbolList.push(getBit(data[1], i, 1)));
         break;
       case 1:
```
Comment (shinyoshiaki) [inline packages/rtp/src/rtcp/rtpfb/twcc.ts:326]: incorrect

File: packages/rtp/src/rtcp/rtpfb/twcc.ts [hunk 2]
```diff
@@ -313,14 +312,20 @@ export class StatusVectorChunk {
...
-        range(3).forEach((i) => symbolList.push(getBit(data[0], 2 + i * 2, 2)));
-        range(4).forEach((i) => symbolList.push(getBit(data[1], i * 2, 2)));
+        range(6, (i) => symbolList.push(getBit(data[0], 2 + i * 2, 2)));
+        range(6, (i) => symbolList.push(getBit(data[1], i * 2, 2)));
```
Comment (shinyoshiaki) [inline packages/rtp/src/rtcp/rtpfb/twcc.ts]: incorrect

**Comment by shinyoshiaki** [comment]
If fix that point, the problem with the test will go away.

File: packages/rtp/src/rtcp/rtpfb/twcc.ts (hunk 2 above)
Comment (koush) [reply packages/rtp/src/rtcp/rtpfb/twcc.ts]: oh whoops sorry, thanks

**Comment by koush** [comment]
> If fix that point, the problem with the test will go away.

Done

**Comment by shinyoshiaki** [comment]
LGTM

**Pull request merged**